lowrisc-toolchain-gcc-multilib-linux.config

```diff
@@ -0,0 +1,40 @@
+CT_CONFIG_VERSION="3"
+CT_EXPERIMENTAL=y
+CT_ARCH_RISCV=y
+CT_ARCH_64=y
+CT_ARCH_USE_MMU=y
+
+# We wish MULTILIB was supported, but it's not - gcc says its built a multilib
+# for rv32, but then glibc fails to build for 32-bit RISC-V. The solution is
+# to not do a multilib build for the moment, but to just match the rv64 default
+# lib, so that later we can add a multilib without breaking everything
+# (hopefully).
+# CT_MULTILIB is not set
+# CT_ARCH_ARCH="rv64gc"
+# CT_ARCH_ABI="lp64d"
+
+# CT_DEMULTILIB is not set
+CT_CC_LANG_CXX=y
+CT_KERNEL_LINUX=y
+CT_DEBUG_GDB=y
+# CT_GDB_CROSS_PYTHON is not set
+# CT_GDB_GDBSERVER is not set
+CT_ZLIB_NEEDED=y
+
+# Disable progress bar for CI builds, it generates too much log output
+# CT_LOG_PROGRESS_BAR is not set
+
+# Installation prefix directory; the toolchain will end up in exactly
+# this directory. CT_PREFIX_DIR will be added by the
+# ./build-gcc-with-args.sh script
+
+# Don't save tarballs for re-use (needs writable and pre-created
+# CT_LOCAL_TARBALLS_DIR otherwise)
+# CT_SAVE_TARBALLS is not set
+
+# Having errors building linux (due to sched.h issues)
```

Comment:
- [ ] These issues might be solved by updating to crosstool-ng HEAD? I think my patch was accepted.